ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cv_pathfinder_flow_tracking.schema.yml

```
@@ -0,0 +1,45 @@
+# Copyright (c) 2023-2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  cv_pathfinder_flow_tracking:
```

We want to use this for more than just cv_pathfinder in the long run
```suggestion
  flow_tracking_settings:
```

---

In that case should we allow the exporter to be a list like in cli gen, or are we ok having only one epxorter.

---

Also the default name should it change from WAN-FLOW-TRACKER